src/background/http-instrument.ts

```diff
@@ -493,10 +504,24 @@ export class HttpInstrument {
     };
     */
 
+    const responseStatus = details.statusCode;
+    const responseStatusText = details.statusLine;
+
```

Comment:
This listener doesn't currently have enough data to track redirects. The data we have available to us in the [details object](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/onBeforeRedirect#details) is:
* `redirectUrl`: The URL that will be used for the new request
* `url`: the URL requested for the previous request which resulted in a `Location` header response.

We can save these as `old_request_url` and `new_request_url`. Ideally we should file a bug on Bugzilla to get `redirectRequestId` added to the Web Extensions API.

Reply:
> We can save these as old_request_url and new_request_url.
Done
> Ideally we should file a bug on Bugzilla to get redirectRequestId added to the Web Extensions API.
Agree